configure.ac

```diff
@@ -1449,7 +1449,7 @@ if test "$use_natpmp" != "no"; then
   CPPFLAGS="$TEMP_CPPFLAGS"
 fi
 
-if test "$build_bitcoin_wallet$build_bitcoin_cli$build_bitcoin_tx$build_bitcoin_util$build_bitcoind$bitcoin_enable_qt$use_tests$use_bench" = "nononononononono"; then
+if test "$build_bitcoin_wallet$build_bitcoin_cli$build_bitcoin_tx$build_bitcoin_util$build_bitcoind$bitcoin_enable_qt$use_tests$use_bench$enable_fuzz" = "nonononononononono"; then
```

Comment:
I wont make much difference, because they will both be set. Changed to `enable_fuzz_binary`.

Reply:
> I wont make much difference

It will. With my testing [setup](https://github.com/bitcoin/bitcoin/pull/27057#pullrequestreview-1289239152) I have an error for 11caed2edbadc5ef2078550c2f323bf9855f5d8d:
```
$ make
Making all in src
make[1]: Entering directory '/home/hebasto/git/bitcoin/src'
make[2]: Entering directory '/home/hebasto/git/bitcoin/src'
  CXX      test/fuzz/fuzz-addition_overflow.o
  CXX      test/fuzz/fuzz-addrman.o
  CXX      test/fuzz/fuzz-asmap.o
  CXX      test/fuzz/fuzz-asmap_direct.o
  CXX      test/fuzz/fuzz-autofile.o
  CXX      test/fuzz/fuzz-banman.o
  CXX      test/fuzz/fuzz-base_encode_decode.o
  CXX      test/fuzz/fuzz-bitdeque.o
  CXX      test/fuzz/fuzz-block.o
  CXX      test/fuzz/fuzz-block_header.o
  CXX      test/fuzz/fuzz-blockfilter.o
  CXX      test/fuzz/fuzz-bloom_filter.o
  CXX      test/fuzz/fuzz-buffered_file.o
  CXX      test/fuzz/fuzz-chain.o
  CXX      test/fuzz/fuzz-checkqueue.o
  CXX      test/fuzz/fuzz-coins_view.o
In file included from ./validation.h:29,
                 from test/fuzz/block.cpp:14:
./txmempool.h:35:10: fatal error: boost/multi_index/hashed_index.hpp: No such file or directory
   35 | #include <boost/multi_index/hashed_index.hpp>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [Makefile:16020: test/fuzz/fuzz-block.o] Error 1
make[2]: *** Waiting for unfinished jobs....
In file included from ./validation.h:29,
                 from test/fuzz/coins_view.cpp:20:
./txmempool.h:35:10: fatal error: boost/multi_index/hashed_index.hpp: No such file or directory
   35 | #include <boost/multi_index/hashed_index.hpp>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [Makefile:16118: test/fuzz/fuzz-coins_view.o] Error 1
make[2]: Leaving directory '/home/hebasto/git/bitcoin/src'
make[1]: *** [Makefile:19229: all-recursive] Error 1
make[1]: Leaving directory '/home/hebasto/git/bitcoin/src'
make: *** [Makefile:815: all-recursive] Error 1
```